modules/renter.go

```diff
@@ -343,6 +343,10 @@ type Renter interface {
 	// ShareFilesAscii creates an ASCII-encoded '.sia' file.
 	ShareFilesASCII(paths []string) (asciiSia string, err error)
 
+	// Streamer creates a io.ReadSeeker that can be used to stream downloads
+	// from the Sia network.
+	Streamer(siaPath string) (string, io.ReadSeeker, error)
```

Comment:
What is the `string` return value? Should either be named or mentioned in the docstring.